web/client/resources/js/goconvey.js

```diff
@@ -769,7 +798,9 @@ function process(data, status, jqxhr)
 	convey.intervalFuncs.momentjs();
 
 	// Show notification, if enabled
-	if (notif())
+	var levelRegex = new RegExp("("+convey.notificationLevel+")", "i");
+	var levelEnabled = current().overall.status.class.match(levelRegex);
```

Comment:
I guess I just like the idea of being able to add a new option or a more specific filtering of goconvey's various states without having to dig through 1.35k lines of javascript to add additional logic. For example, I get the benefit of a build failure and a test failure by using the regex string method.